Makefile

```diff
@@ -1,6 +1,6 @@
-.PHONY: clean package test
+.PHONY: build check-env clean delete-stack package rebuild test
 
-all: build test package
+all: package
```

Comment:
`make` should build `all
which should build `./packaged.yaml`
which should build `./template.yaml`

Although, it will only do so when they don't already exist or when they do exist but `src/aws_budget_alerting.py` alerting has been changed since they were created.

If you want to see it create `./template.yaml` you might need to do a `make clean` before hand.

Reply:
dependencies updated as discussed with @daniel-ac-martin